lnbits/core/crud.py

```diff
@@ -412,7 +413,8 @@ async def get_payments(
         FROM apipayments
         {where}
         ORDER BY time DESC
-        {limit_offset_clause}
+        {limit_clause}
+        {offset_clause}
```

Comment:
Sure this works without a space in between?

Reply:
this is completly refactored inside this PR https://github.com/lnbits/lnbits/pull/1557